references/validationFunctions/primary_diagnosis/stageGroup.js

```diff
@@ -28,7 +28,12 @@ const validation = () =>
   (function validate(inputs) {
     const {$row, $name, $field} = inputs;
     let result = { valid: true, message: 'Ok' };
-    if ($row.clinical_tumour_staging_system && $field) {
+    if (!($row.clinical_tumour_staging_system) && $field) {
+       result.valid = false;
+       const msg = "'clinical_tumour_staging_system' must be submitted if 'clinical_stage_group' is submitted";
```

Comment:
Friendlier error for ui: 
`The field` 'clinical_tumour_staging_system' must be submitted if 'clinical_stage_group' is submitted"`.`